<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathMatrixUi</class>
 <widget class="QDialog" name="MathMatrixUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="rowsL">
       <property name="toolTip">
        <string>Number of rows</string>
       </property>
       <property name="text">
        <string>&amp;Rows:</string>
       </property>
       <property name="buddy">
        <cstring>rowsSB</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rowsSB">
       <property name="toolTip">
        <string>Number of rows</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>511</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="columnsL">
       <property name="toolTip">
        <string>Number of columns</string>
       </property>
       <property name="text">
        <string>&amp;Columns:</string>
       </property>
       <property name="buddy">
        <cstring>columnsSB</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="columnsSB">
       <property name="toolTip">
        <string>Number of columns</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>511</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="EmptyTable" name="table" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Resize this to the correct table dimensions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="alignmentGB">
     <property name="title">
      <string>Alignment</string>
     </property>
     <layout class="QGridLayout" name="grid_1">
      <property name="margin">
       <number>11</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="1" column="0">
       <widget class="QComboBox" name="valignCO">
        <property name="toolTip">
         <string>Vertical alignment</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="valignLA">
        <property name="text">
         <string>&amp;Vertical:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>valignCO</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="halignED">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Horizontal alignment per column (l,c,r)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="widthLA_2">
        <property name="text">
         <string>&amp;Horizontal:</string>
        </property>
        <property name="buddy">
         <cstring>halignED</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QGroupBox" name="decorationtGB">
     <property name="title">
      <string>Decoration</string>
     </property>
     <layout class="QGridLayout" name="grid_2">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLA">
        <property name="text">
         <string>&amp;Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>decorationCO</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="decorationCO">
        <property name="toolTip">
         <string>decoration type / matrix border</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EmptyTable</class>
   <extends>QWidget</extends>
   <header>EmptyTable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rowsSB</tabstop>
  <tabstop>columnsSB</tabstop>
  <tabstop>valignCO</tabstop>
  <tabstop>halignED</tabstop>
  <tabstop>decorationCO</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
